data/src/main/java/io/homeassistant/companion/android/data/url/UrlRepositoryImpl.kt

```diff
@@ -28,6 +28,11 @@ class UrlRepositoryImpl @Inject constructor(
         val retVal = ArrayList<URL>()
         val webhook = localStorage.getString(PREF_WEBHOOK_ID)
 
+        // If we don't have a webhook id we don't have any urls.
```

Comment:
This is technically correct. But can't we abort onboarding if it fails ? And what happens if we return an empty array here?

Reply:
The check for an empty apiUrl list is already in place during registration, this defect is a regression.
Check for is registered: https://github.com/home-assistant/home-assistant-android/blob/master/app/src/main/java/io/homeassistant/companion/android/launch/LaunchPresenterImpl.kt#L33

isRegistered check: https://github.com/home-assistant/home-assistant-android/blob/master/data/src/main/java/io/homeassistant/companion/android/data/integration/IntegrationRepositoryImpl.kt#L122